vardbg/output/video_writer/config.py

```diff
@@ -106,7 +106,9 @@ def __init__(self, config_path):
         self.font_heading = (sub_path(fonts["heading"]), fonts["heading_size"])
         self.font_intro = (sub_path(fonts["intro"]), fonts["intro_size"])
 
-        style = MonokaiStyle
+        theme = self.data["theme"]
+        color_scheme_name = theme["color_scheme"]
+        style = get_style_by_name(color_scheme_name)
```

Comment:
```suggestion
        style = get_style_by_name(theme["color_scheme"])
```

No need to define a new variable for a quick thing that's only used once.

Reply:
Okay I will update this suggestion.